src/main/java/com/vmware/vipclient/i18n/messages/service/PatternService.java

```diff
@@ -30,63 +33,90 @@ public JSONObject getPatternsByCategory(String locale, String category) {
 
     public JSONObject getPatterns(String locale) {
```

Comment:
I tried, but failed.
1. The cache content's type of language and region is Map<String, String>, but pattern's type is Map<String, Object>,
the Map<String, String> in FormatCacheItem can't  be compatible with these two types.
2. The structure of pattern data is nested deeply, and used in many places, it will take a lot of time to change all references.